pkg/oonf_api/patches/0002-port-tests-to-riot.patch

```diff
@@ -139,14 +142,16 @@ index 0000000..e472545
 +export BOARD ?= native
 +
 +# this has to be the absolute path of the RIOT-base dir
-+export RIOTBASE =$(CURDIR)/../../../../../..
++export RIOTBASE =$(CURDIR)/../../../../../../../../..
 +
 +CFLAGS = -DOONF_LOG_INFO -DOONF_LOG_DEBUG_INFO
++WERROR = 0
```

Comment:
What's the reason for this?

Reply:
Because of 
```C
In file included from /home/benpicco/dev/RIOT/tests/pkg_oonf_api/bin/pkg/native/oonf_api/tests/common/test_common_avl/bin/pkg/native/oonf_api/tests/cunit/cunit.c:45:
/home/benpicco/dev/RIOT/tests/pkg_oonf_api/bin/pkg/native/oonf_api/tests/common/test_common_avl/bin/pkg/native/oonf_api/tests/cunit/cunit.h:59:56: warning: ISO C does not permit named variadic macros [-Wvariadic-macros]
 #define CHECK_NAMED_TRUE(cond, name, line, format, args...) cunit_named_check(cond, name, line, format, ##args);
                                                        ^~~
/home/benpicco/dev/RIOT/tests/pkg_oonf_api/bin/pkg/native/oonf_api/tests/common/test_common_avl/bin/pkg/native/oonf_api/tests/cunit/cunit.h:60:38: warning: ISO C does not permit named variadic macros [-Wvariadic-macros]
 #define CHECK_TRUE(cond, format, args...) cunit_named_check(cond, __func__, __LINE__, format, ##args);
                                      ^~~

In file included from /home/benpicco/dev/RIOT/tests/pkg_oonf_api/bin/pkg/native/oonf_api/tests/common/test_common_avl/test_common_avl.c:48:
/home/benpicco/dev/RIOT/tests/pkg_oonf_api/bin/pkg/native/oonf_api/tests/common/test_common_avl/../../cunit/cunit.h:59:56: warning: ISO C does not permit named variadic macros [-Wvariadic-macros]
 #define CHECK_NAMED_TRUE(cond, name, line, format, args...) cunit_named_check(cond, name, line, format, ##args);
                                                        ^~~
/home/benpicco/dev/RIOT/tests/pkg_oonf_api/bin/pkg/native/oonf_api/tests/common/test_common_avl/../../cunit/cunit.h:60:38: warning: ISO C does not permit named variadic macros [-Wvariadic-macros]
 #define CHECK_TRUE(cond, format, args...) cunit_named_check(cond, __func__, __LINE__, format, ##args);
                                      ^~~

/home/benpicco/dev/RIOT/sys/posix/sockets/posix_sockets.c: In function ‘socket_recvfrom’:
/home/benpicco/dev/RIOT/sys/posix/sockets/posix_sockets.c:814:60: warning: unused parameter ‘buffer’ [-Wunused-parameter]
 static ssize_t socket_recvfrom(socket_t *s, void *restrict buffer,
                                             ~~~~~~~~~~~~~~~^~~~~~
/home/benpicco/dev/RIOT/sys/posix/sockets/posix_sockets.c:815:39: warning: unused parameter ‘length’ [-Wunused-parameter]
                                size_t length, int flags,
                                ~~~~~~~^~~~~~
/home/benpicco/dev/RIOT/sys/posix/sockets/posix_sockets.c: In function ‘socket_sendto’:
/home/benpicco/dev/RIOT/sys/posix/sockets/posix_sockets.c:909:55: warning: unused parameter ‘buffer’ [-Wunused-parameter]
 static ssize_t socket_sendto(socket_t *s, const void *buffer, size_t length,
                                           ~~~~~~~~~~~~^~~~~~
/home/benpicco/dev/RIOT/sys/posix/sockets/posix_sockets.c:909:70: warning: unused parameter ‘length’ [-Wunused-parameter]
 static ssize_t socket_sendto(socket_t *s, const void *buffer, size_t length,
                                                               ~~~~~~~^~~~~~
/home/benpicco/dev/RIOT/sys/posix/sockets/posix_sockets.c:910:64: warning: unused parameter ‘address’ [-Wunused-parameter]
                              int flags, const struct sockaddr *address,
                                         ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~
/home/benpicco/dev/RIOT/sys/posix/sockets/posix_sockets.c:911:40: warning: unused parameter ‘address_len’ [-Wunused-parameter]
                              socklen_t address_len)
                              ~~~~~~~~~~^~~~~~~~~~~
```

Reply:
I must say I'm getting fond of the idea of helping you with your project in reducing disc space that RIOT takes up by deprecating this pkg.

Nobody cared enough about it to update it or run tests in 6 years and with nhdp being deprecated, it's only user is on it's way out too.

And any downstream would probably prefer a less vintage version of the library. (Or should we rather just call it stable 😏 )